### ISSUE TITLE
Unison Share requires a Microsoft GitHub account

### ISSUE THREAD

**Issue by toastal** [unisonweb/unison#3590]
I don't think it's acceptable to tie the service's identity and by proxy any contributions with an account with a closed-source, proprietary service. This also leads to vendor lock-in that may come back to bite the project.

To respect the freedom and privacy of users, I recommend opening this up to other, preferably decentralized, services for identity.

**Comment by mitchellwrosen** [comment]
Indeed! We intend to open this up to more identity providers eventually.
